src/engine/runtime.js

```diff
@@ -1012,6 +1013,51 @@ class Runtime extends EventEmitter {
             this._step();
         }, interval);
     }
+
+    // ---------------------------------------------------------------------
```

Comment:
copy pasta from:
L81
https://github.com/LLK/scratch-vm/blob/develop/src/engine/blocks.js

Should I not follow this style?